src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.cpp

```diff
@@ -1923,12 +1928,20 @@ CHIP_ERROR GenericThreadStackManagerImpl_OpenThread<ImplClass>::SEDChangeMode()
 }
 
 template <class ImplClass>
-void GenericThreadStackManagerImpl_OpenThread<ImplClass>::RequestSEDModeChange(chip::System::Layer * apSystemLayer,
+void GenericThreadStackManagerImpl_OpenThread<ImplClass>::RequestSEDModeUpdate(chip::System::Layer * apSystemLayer,
                                                                                void * apAppState)
 {
     if (apAppState != nullptr)
     {
-        static_cast<GenericThreadStackManagerImpl_OpenThread *>(apAppState)->SEDChangeMode();
+        GenericThreadStackManagerImpl_OpenThread * obj = static_cast<GenericThreadStackManagerImpl_OpenThread *>(apAppState);
+        if (obj->mActiveModeConsumers > 0)
+        {
+            obj->mActiveModeConsumers--;
+        }
```

Comment:
for sanity, should we have a log if active consumers is 0 and this is triggered since it sounds like an internal error?

Reply:
Yes and No. The API is public, hence it can be called from a client/user App layer in which we have no control. Adding a log here in the Matter stack could be misleading. I would leave it as is since this scenario is handled properly.

A log for : "You're doing something wrong, but it won't break anything" will just contribute to the overall bloat of Matter IMO